modules/circe/src/main/scala/pureconfig/module/circe/package.scala

```diff
@@ -0,0 +1,57 @@
+package pureconfig.module
+
+import pureconfig.{ ConfigWriter, ConfigReader }
+import com.typesafe.config._
+import io.circe._
+import com.typesafe.config._
+
+import scala.collection.JavaConverters._
+
+package object circe {
+  def cvToJson(cv: ConfigValue): Json = {
+    cv.valueType match {
+      case ConfigValueType.NULL =>
+        Json.Null
+      case ConfigValueType.BOOLEAN =>
+        Json.fromBoolean(cv.unwrapped.asInstanceOf[Boolean])
+      case ConfigValueType.NUMBER =>
+        cv.unwrapped match {
+          case i: java.lang.Number if i.longValue() == i =>
+            Json.fromLong(i.longValue())
+          case d: java.lang.Number if !(d.doubleValue().isNaN) =>
+            Json.fromDouble(d.doubleValue()).get
+          case _ => // Non-integral, non-real; following javascript's JSON.stringify in gives null
+            Json.Null
```

Comment:
I'm sure there's strong opinions about this one way or the other but I decided to just follow JS behavior here as a first pass

Reply:
I'm OK with `Json.Null` here. It is consistent with what the default `Encoder` for `Double` does in circe. However, we also need to take infinity into account. I think we can remove this fallback and use `Json.fromDoubleOrNull` instead.